### ISSUE TITLE
Obfuscated source

### ISSUE THREAD

**Issue by bpatkins** [ClimberAB/ClimberCustomReport#19]
Is it possible to get the code before it was minified? Trying to extend this a bit and it would be a good resource. I can format it easily enough but the cl-customreport.js file is a bit hard to follow with the a-o injections.

Thanks! 

![report qlik](https://cloud.githubusercontent.com/assets/3619077/25281512/4696cd64-2673-11e7-876c-bef2e5337c34.PNG)

**Comment by ClimberExtensions** [comment]
Hi!
We had lot's of users that were confused by the multiple folders and code on GitHub when they downloaded and installed the extensions. Therefor (and since we had no contributions for any of our projects so far ;-)) we decided to keep it simple and only share the code people use. 

/Michael
